lang/ja/locales.php

```diff
@@ -3,4 +3,6 @@
 return [
     'en' => '英語',
     'ja' => '日本語',
+    'zh_CN' => '簡体字中国語',
+    'zh_TW' => '繁体字中国語',
```

Comment:
This sounds more natural I think ↓
```suggestion
    'zh_CN' => '簡体中国語',
    'zh_TW' => '繁体中国語',
```

Reply:
done.